features/supplier/brief_response.feature

```diff
@@ -3,238 +3,279 @@ Feature: Supplier applies for a brief
+  And I go to that brief page
+  And I click 'Apply'
+  Then I am on 'You can’t apply for this opportunity' page
+  And I see 'You can’t apply for this opportunity because you didn’t say you could provide services in this category when you applied to the Digital Outcomes and Specialists 2 framework.' text on the page
+  And I see a 'data-reason' attribute with the value 'supplier-not-on-lot'
+
+@not-eligible-for-brief-response
+Scenario: Supplier is not eligible as they can not provide the developer role
+  Given that supplier is on that framework
+  And that supplier has a service on the digital-specialists lot for the designer role
+  And I go to that brief page
+  And I go to that brief page
+  And I click 'Apply'
+  Then I am on 'You can’t apply for this opportunity' page
+  And I see 'You can’t apply for this opportunity because you didn’t say you could provide this specialist role when you applied to the Digital Outcomes and Specialists 2 framework.' text on the page
+  And I see a 'data-reason' attribute with the value 'supplier-not-on-role'
+
+@eligible-for-brief-response
 Scenario: Supplier applies for a digital-specialists brief
-  Given that supplier has a service on the digital-specialists lot
-    And I have a live digital-specialists brief
-    And I go to that brief page
-    And I click 'Apply'
-    Then I am on 'Apply for ‘Tea drinker’' page
+  Given that supplier is on that framework
+  And that supplier has a service on the digital-specialists lot
+  And I have a live digital-specialists brief
+  And I go to that brief page
+  And I click 'Apply'
+  Then I am on 'Apply for ‘Tea drinker’' page
   When I click 'Start application'
-    Then I am on 'When is the earliest the specialist can start work?' page
-    And I see 'The buyer needs the specialist to start: 31/12/2016' replayed in the question advice
+  Then I am on 'When is the earliest the specialist can start work?' page
+  And I see 'The buyer needs the specialist to start: 31/12/2016' replayed in the question advice
   When I enter '27/12/17' in the 'availability' field
-    And I click 'Continue'
-    Then I am on the 'What’s the specialist’s day rate?' page
-    And I see '£200' replayed in the question advice
+  And I click 'Continue'
+  Then I am on the 'What’s the specialist’s day rate?' page
+  And I see '£200' replayed in the question advice
   When I enter '200' in the 'dayRate' field
-    And I click 'Continue'
-    Then I am on the 'Do you have all the essential skills and experience?' page
+  And I click 'Continue'
+  Then I am on the 'Do you have all the essential skills and experience?' page
   When I choose 'Yes' radio button
-    And I click 'Continue'
-    Then I am on the 'Give evidence of the essential skills and experience' page
+  And I click 'Continue'
+  Then I am on the 'Give evidence of the essential skills and experience' page
   When I enter 'first evidence' in the 'Boil kettle' field
-    And I enter 'second evidence' in the 'Taste tea' field
```

Comment:
I look forward to the next PR where all these whitespace changes are reverted to align the `I`s :smile: